openghg/standardise/surface/_eurocom.py

```diff
@@ -89,7 +81,7 @@ def date_parser(year: str, month: str, day: str, hour: str, minute: str) -> Time
         data_filepath,
         skiprows=n_skip,
         parse_dates=datetime_columns,
-        date_parser=date_parser,
+        date_format="%Y %m %d %H %M",
```

Comment:
Just to check, for the new `date_format` input this is able to convert a set of separate columns into a single datetime using a `str` input like this (instead of e.g. dict of column)?

I can believe it works like that but the documentation isn't clear so wanted to check the new pandas 2.0 functionality:

> date_format  str or dict of column -> format, optional
> 
 >   Format to use for parsing dates when used in conjunction with parse_dates. For anything more complex, please read in as object and then apply [to_datetime()](https://pandas.pydata.org/docs/reference/api/pandas.to_datetime.html#pandas.to_datetime) as-needed.

Reply:
The documentation isn't very useful... it's a bit better if you look at the docs for the dev branch. What is happening is that `parse_dates` is making a new column where the date columns are joined by spaces, and then `date_format` is parsing this new column. I'm not totally sure how `date_format` knows which column to parse... I think it just tries to parse each column until it succeeds, although maybe there is a way to be more specific.

Reply:
Ok great, thanks for the details. I guess it maybe just parses the newly named columns created from `parse_dates` input. Anyway that sounds good to me.